tools/releasing/packaging/Arch_User_Repository/PKGBUILD

```
@@ -1,29 +1,37 @@
 # Maintainer:  Florian Lindner <florian.lindner@xgm.de>
 
 pkgname=precice
-pkgver=1.6.1
+pkgver=2.1.0
```

```suggestion
pkgver=2.1.1
```
or is this a post release action?